src/app/containers/FrontPageMain/index.jsx

```diff
@@ -2,19 +2,21 @@ import React, { Fragment } from 'react';
 import { shape } from 'prop-types';
 import { articleDataPropTypes } from '../../models/propTypes/article';
 import deepGet from '../../helpers/json/deepGet';
-import { GhostWrapper } from '../../lib/styledGrid';
+import { GhostWrapper, GridItemConstrainedLarge } from '../../lib/styledGrid';
+import FrontPageSection from '../FrontPageSection';
 
 const FrontPageMain = ({ frontPageData }) => {
-  const title = deepGet(
-    ['content', 'groups', 0, 'items', 0, 'headlines', 'headline'],
-    frontPageData,
-  );
+  const content = deepGet(['content', 'groups'], frontPageData);
 
   return (
     <Fragment>
       <main role="main">
         <GhostWrapper>
-          <h1>{title}</h1>
+          <GridItemConstrainedLarge>
+            {content.map(group => (
```

Comment:
Not checked yet, but this likely needs a key

Reply:
Yeah, bunch of warnings in storybook, can you try and resolve them please?

Reply:
Most of them are because of the comment I left at the bottom of the PR